README.md

```diff
@@ -143,6 +117,87 @@ In order to start up oneview-redfish-toolkit service, there is some mandatory co
  
   * **emailAddress**: Email address to contact the responsible for this server/certificate. This is an optional information. Will not be added to certificate if not informed. **Optional.**
 
+## Logging
+
+Logging configuration can be found in `logging.conf` file. The provided configuration enables INFO level at both console and file output, which will generate a `redfish.log` file at `$HOME/.config/oneview-redfish-toolkit/`.
```

Comment:
we should reference the other log files here, too.

Reply:
Done.